packages/boot/src/interfaces.ts

```diff
@@ -60,10 +57,10 @@ export const BOOTER_PHASES = ['configure', 'discover', 'load'];
 /**
  * Type Object for Options passed into .boot()
  *
- * @property projectRoot Root of project. All other artifacts are resolved relative to this
- * @property booters An array of booters to bind to the application before running bootstrapper
- * @property filter.booters An array of booters that should be run by the bootstrapper
- * @property filter.phases An array of phases that should be run
+ * - projectRoot: Root of project. All other artifacts are resolved relative to this
+ * - booters: An array of booters to bind to the application before running bootstrapper
+ * - filter.booters: An array of booters that should be run by the bootstrapper
+ * - filter.phases: An array of phases that should be run
```

Comment:
This documentation does not match the described interface.

I believe `booters` and `filters` properties were moved from `BootOptions` to `BootExecutionOptions`; and `projectRoot` was moved to `Bootable` interface.

Reply:
This can be fixed in a follow-up PR. The original doc follows js style. For TS, such properties should be documented directly at  `BootExecutionOptions` and `Bootable` level.